xbmc/addons/Service.h

```diff
@@ -71,10 +56,15 @@ namespace ADDON
     void Start(const std::string& addonId);
 
     /**
-     * Stop all services.
+     * Stop the service addon manager and all services.
      */
     void Stop();
 
+    /**
+     * Stop all services.
+     */
+    void StopServices();
```

Comment:
Why is there two start and stop methods? If it's just the un-/subscribing it's fine to do this every profile change